content/open-source/melange/tutorials/getting-started-with-melange.md

```diff
@@ -361,15 +361,16 @@ The following command will set up a volume sharing your current folder with the
 ```shell
 docker run --rm -v ${PWD}:/work cgr.dev/chainguard/apko \
   build apko.yaml hello-minicli:test hello-minicli.tar \
+  --arch host
   -k melange.rsa.pub
```

Comment:
I ran into an issue with this command. For some reason (at least in the preview) it treats the last line of this command (`-k melange.rsa.pub`) as a new command, so it doesn't work. 

```
. . .
Error: failed to build layer image for "amd64": installing apk packages: error getting package dependencies: error getting repository indexes: no key found to verify signature for keyfile melange.rsa.pub; tried all other keys as well
2023/08/04 16:40:37 error during command execution: failed to build layer image for "amd64": installing apk packages: error getting package dependencies: error getting repository indexes: no key found to verify signature for keyfile melange.rsa.pub; tried all other keys as well
-k: command not found
```

I did run it as a single line (`docker run --rm -v ${PWD}:/work cgr.dev/chainguard/apko   build apko.yaml hello-minicli:test hello-minicli.tar   --arch host -k melange.rsa.pub`) and it worked a-okay.

Reply:
Ah yeah, nice catch, missed the `\` at the end of the `--arch host` line